index.html

```diff
@@ -22,7 +22,7 @@
       <div class="input-group input-group-lg">
         <input type="text" class="form-control" name="q" placeholder="Search ROS Packages">
         <span class="input-group-btn">
-          <button class="btn btn-default btn-lg" type="submit"><span class="glyphicon glyphicon-search"></span></button>
+          <button class="btn btn-default btn-lg" type="submit" title="Search ROS Packages"><span class="glyphicon glyphicon-search"></span></button>
```

Comment:
Note that this one is inconsistent with the others, which have been changed to say "Find in rosindex". I don't have a preference among the two, but I think it would be nice to be consistent.

Reply:
Good point, I renamed all them to show "Search ROS Packages" again.